modules/consensus/database.go

```diff
@@ -21,7 +23,7 @@ var (
 	errNonEmptyBucket = errors.New("cannot remove a map with objects still in it")
 
 	dbMetadata = persist.Metadata{
-		Version: "0.4.3",
+		Version: "0.5.0",
```

Comment:
This should only be bumped if we are introducing breaking changes the to the DB. Is that the case?